.github/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+<!--
+Thanks for sending a pull request!  Here are some tips for you:
+  1. If this is your first time, please read our contributor guidelines: https://github.com/delta-incubator/delta-kernel-rs/blob/main/CONTRIBUTING.md
+  2. Ensure you have added or run the appropriate tests for your PR
+  3. If the PR is unfinished, add '[WIP]' in your PR title, e.g., '[WIP] Your PR title ...'.
+  4. Be sure to keep the PR description updated to reflect all changes.
+  5. Please write your PR title to summarize what this PR proposes.
+  6. If possible, provide a concise example to reproduce the issue for a faster review.
+-->
+
+### What changes were proposed in this pull request?
+<!--
+Please clarify what changes you are proposing. The purpose of this section is to outline the changes and how this PR fixes the issue.
+If possible, please consider writing useful notes for better and faster reviews in your PR.
+-->
+
+
+### Why are the changes needed?
+<!--
+Please clarify why the changes are needed. If this is already explained clearly in an issue, then this section is not needed.
+  1. If you propose a new API or feature, clarify the use case for a new API or feature.
+  2. If you fix a bug, you can clarify why it is a bug.
+-->
+
+
+### Does this PR introduce _any_ user-facing change?
+<!--
+Note that it means *any* user-facing change including all aspects such as the documentation fix.
+If yes, please clarify the previous behavior and the change this PR proposes - provide the console output, description and/or an example to show the behavior difference if possible.
+If no, write 'No'.
+-->
+
+
+### How was this change tested?
+<!--
+If tests were added, say they were added here. Please make sure to add some test cases that check the changes thoroughly including negative and positive cases if possible.
+If it was tested in a way different from regular unit tests, please clarify how you tested step by step, ideally copy and paste-able, so that other reviewers can test and check, and descendants can verify in the future.
+If tests were not added, please describe why they were not added and/or why it was difficult to add.
+-->
+
+
+<!--
+Adapted from Apache Spark's pull request template: https://github.com/apache/spark
```

Comment:
Do we need this?

Reply:
i think this was originally my suggestion :) but i meant for us to just credit them in the PR description or something - don't need to have it in the actual template